starboard/shared/widevine/drm_system_widevine.cc

```diff
@@ -235,9 +236,10 @@ DrmSystemWidevine::DrmSystemWidevine(
   }
 #endif  // !defined(COBALT_BUILD_TYPE_GOLD)
 
-  EnsureWidevineCdmIsInitialized(company_name, model_name);
+  static WidevineStorage s_storage(GetWidevineStoragePath());
```

Comment:
My questions in C++ are often remedial, but—
is this moved here from line 162 above mainly
to ensure it's not used on lines 193–194?
If so, does that scoping issue interact with
the question of making it a class member?

Reply:
@kaidokert We could, but I figured to keep it out since the scope is limited. It'll only be needed for the constructor along with the global `EnsureWidevineCdmIsInitialized()`.

@dahlstrom-g This line is moved so I can pass `s_storage` to `EnsureWidevineCdmIsInitialized()` while also using it in `wv3cdm::create()`. We could make it a class member, just I don't think the class needs to hold onto it.

Reply:
Oops; I missed that it was moved from a function
to a constructor (whose definition started in the fold).
I have a hard time with the GitHub UI. 😬 Thanks!